src/components/SidePanel/styles.module.scss

```diff
@@ -22,9 +22,15 @@
 }
 
 .content {
+  border-radius: $border-radius;
+  border: 1px solid $light-gray;
```

Comment:
I think this introduced an extra border around the collapsed panel (which shouldn't be there). Can you take a look?

<details>
<summary>screenshot</summary>

<img width="235" alt="Screenshot 2019-07-01 17 01 00" src="https://user-images.githubusercontent.com/55398/60469375-03ea0400-9c22-11e9-9b1d-47d45def6e45.png">


</details>